src/components/vega/ScheduleViz.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useLayoutEffect } from "react";
+import vegaEmbed from "vega-embed";
+import spec from "../../assets/vega/schedule";
+import data from "../../assets/vega/sched_viz_data";
+
+const ScheduleViz = (props) => {
+  const [view, setView] = useState(null);
+  const [rendered, setRendered] = useState(false);
+
+  useLayoutEffect(() => {
+    if (!rendered) {
+      vegaEmbed("#schedule", spec, {
```

Comment:
Typically, querying things by id is a ~~bad thing~~ in modern web dev. Honestly, it's fine though. Happy to chat "offline" about what the recommended way to do things is, but I don't want to get overly pedantic here. :^)